src/babylon.scene.ts

```diff
@@ -1382,6 +1382,17 @@
             return this._boundingBoxRenderer;
         }
 
+        private _utilityLayerRenderer:UtilityLayerRenderer;
+        /** 
+         * Gets the UtilityLayerRenderer used to render an overlay scene layer without impacting the existing scene
+         */
+        public get utilityLayerRenderer(): UtilityLayerRenderer {

```

Comment:
Are you sure adding this is a good idea? Having this as a singleton on the scene may make developers mix too many things in the same utility layer. I would want to encourage multiple layers (eg. one for gizmo, one for gui, one for controllers, etc.)

Reply:
this makes sense as well. I like the idea of multiple layer actually. Keep it like that then